scripts/fetch-webhapp.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const fs = require('fs');
+const path = require('path');
+const crypto = require('crypto');
+const tsNode = require('ts-node');
+const childProcess = require('child_process');
+
+tsNode.register();
+
+const kangarooConfig = require(path.join(process.cwd(), 'kangaroo.config.ts')).default;
+
+function downloadFile(url, targetPath, expectedSha256Hex, chmod = false) {
```

Comment:
Do we want to move this into a common file imported here and by `fetch-binaries.js` as well?

Reply:
:+1:  - addressed: fcd483554020ac8a2cf3dd75c63f61ebf1603296